src/internal/components/pluggableVisualizations/columnChart/tests/PluggableColumnChart.spec.tsx

```diff
@@ -126,34 +125,5 @@ describe("PluggableColumnChart", () => {
             expect(measures).toEqual(["m3", "m4"]);
             expect(axis).toEqual(AXIS.DUAL);
         });
-
-        it("should update supported properties list base on axis type", async () => {
```

Comment:
Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-38fc01393275f995733e0ce7e6101b46R229